custom/onse/tasks.py

```diff
@@ -208,6 +209,11 @@ def five_years_ago():
     return datetime.utcnow().date() - relativedelta(years=5)
 
 
+@apply_request_backoff(
+    max_retries=MAX_RETRIES,
+    exponential_base=EXPONENTIAL_BASE,
+    exponential_period=EXPONENTIAL_PERIOD
+)
 def fetch_data_set(
```

Comment:
Are there other important places requests made to DHIS that also needs to be decorated that I might have missed?